src/wp-includes/blocks.php

```diff
@@ -207,6 +207,7 @@ function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
 		'supports'        => 'supports',
 		'styles'          => 'styles',
 		'example'         => 'example',
+		'apiVersion'      => 'api_version',
```

Comment:
It would be great to include this new property in the existing unit test, somewhere around:

https://github.com/gziolo/wordpress-develop/blob/3785439c8bf01ad18727433f3586dc2cbf641e2a/tests/phpunit/tests/blocks/register.php#L282-L288

Test fixture to modify:
https://github.com/gziolo/wordpress-develop/blob/master/tests/phpunit/tests/blocks/fixtures/block.json